apps/explorer/src/components/transactions/TxCardUtils.tsx

```diff
@@ -20,6 +20,7 @@ import styles from './RecentTxCard.module.css';
 
 import { AddressLink, TransactionLink } from '~/ui/InternalLink';
 
+/** @deprecated This styling is not correct, use `<Text>` to create the correctly styled components */
```

Comment:
might be nice to add a ticket to the backlog for removing the remaining usages of `SuiAmount` so it's tracked somewhere 😄